components/dash-core-components/src/components/Input.react.js

```diff
@@ -61,18 +61,22 @@ export default class Input extends PureComponent {
         const valprops =
             this.props.type === 'number' ? {} : {value: this.state.value};
         const {loading_state} = this.props;
+        let {className} = this.props;
+        className = 'dash-input' + (className ? ` ${className}` : '');
         return (
             <input
                 data-dash-is-loading={
                     (loading_state && loading_state.is_loading) || undefined
                 }
+                className={className}
                 ref={this.input}
                 onBlur={this.onBlur}
                 onChange={this.onChange}
                 onKeyPress={this.onKeyPress}
                 {...valprops}
                 {...omit(
                     [
+                        'className',
```

Comment:
FYI we use this a lot but I'd love to get rid of this `...omit` pattern pretty much wherever we use it. It causes various problems, like I bet we're currently improperly forwarding `persistence`/`persistence_type`/`persisted_props` here, and some other places we still forward `loading_state` so the DOM ends up with `loading_state="[object Object]". Instead we should figure out which props we DO want and [pick](https://ramdajs.com/docs/#pick) them, or just individually forward them. I thought I had made an issue for this but not seeing it now 🤔